src/bots/bot.ts

```diff
@@ -65,30 +64,24 @@ export class Bot implements IBot {
   // eslint-disable-next-line max-params
   constructor (botConfig: BotConfig) {
     if(typeof botConfig.credentials !== 'undefined' && botConfig.urls.length === 0) {
-      throw new Error('Bots with credentials cannot have urls as the Bot will never be selected otherwise')
+      throw new Error('Bots with credentials must have atleast one URL, as the Bot will never be selected otherwise')
     }
     this.username = botConfig.credentials?.username
     this.password = botConfig.credentials?.password
     this.urls = botConfig.urls
     if(typeof botConfig.rateLimit === 'number') {
-      // eslint-disable-next-line no-warning-comments
-      // TODO, calculate number of times account was used per minute
       this.rateLimit = botConfig.rateLimit ?? 0
     }
     this.browserConfig = botConfig.browserConfig
     this.usageTimeStamps = []
   }
 
-  private setUsageTimeStamps (): void {
+  public wasUsed (): void {
     const now = Date.now()
     this.usageTimeStamps.push(now)
     this.usageTimeStamps = this.usageTimeStamps.filter(ts => (now - ts) <= ONE_MINUTE)
   }
```

Comment:
What is this method doing? Why does it filter out all timestamps that are older than one minute? Needs tsdocs